pkg/config/store.go

```diff
@@ -78,6 +78,10 @@ type ChangeNotifier interface {
 	// The client should issue ReadChangeLog to see what has changed if the call is available.
 	// else it should re-read the store, perform diff and apply changes.
 	RegisterStoreChangeListener(s StoreListener)
+
+	// IsStoreChangeAvailable returns true only when the backend store is configured
+	// to emit change notifications.
+	IsStoreChangeAvailable() bool
```

Comment:
Basically, because:
```go
var _ store.ChangeNotifier = &redisStore{}
var _ store.ChangeLogReader = &redisStore{}
```
in server:
```go
redis := redis.newStore(...)
listener := cfg.NewListener(redis) // redis as store.ChangeLogReader, returning a store listener
redis.RegisterListener(listener) // redis as store.ChangeNotifier
```